administration/src/application/components/ApplyController.tsx

```diff
@@ -86,6 +100,10 @@ const ApplyController = () => {
             Über den Fortschritt Ihres Antrags werden Sie per E-Mail informiert.
             Sie können das Fenster jetzt schließen.`
 
+  if (!regions) {
+    return null
+  }
```

Comment:
I guess we should show a loading spinner if it's loading and an error comp. if it's erroneous.

Reply:
sure it was just a draft :)

Reply:
added loadingSpinner and refetch